app/controllers/openid_connect/user_info_controller.rb

```diff
@@ -0,0 +1,38 @@
+module OpenidConnect
+  class UserInfoController < ApplicationController
+    skip_before_action :verify_authenticity_token
+    before_action :authenticate_identity_via_bearer_token
+
+    attr_reader :current_identity
+
+    def show
+      render json: OpenidConnectUserInfoPresenter.new(current_identity).user_info
+    end
+
+    private
+
+    def authenticate_identity_via_bearer_token
```

Comment:
What do you think about moving this logic to a separate class that validates the header, and returns a FormResponse, to adhere to our controller design principles, and to be able to track analytics?